speechbrain/dataio/dataio.py

```diff
@@ -210,7 +257,7 @@ def read_audio_info(path) -> "torchaudio.backend.common.AudioMetaData":
     # double-checking anyway. If I am wrong and you are reading this comment
     # because of it: sorry
     if info.num_frames == 0:
-        channels_data, sample_rate = torchaudio.load(path, normalize=False)
+        channels_data, sample_rate = load_audio_pyav(path, normalize=False)
```

Comment:
If using PyAV directly most of this function can be replaced, and reading the entire audio should no longer be necessary. We probably should make it return our own metadata dataclass rather than returning the library's own metadata, though.

Reply:
What do you mean?

Reply:
Most of this function was written to work around `torchaudio.info` being broken with certain mp3 files. Since we're moving to PyAV, the workarounds become unnecessary. In any case, Peter was probably already aware this function should be rewritten with PyAV anyway.

What I mean by the second sentence is that IMO we should make our own dataclass with the same fields as `torchaudio.info` provided for compatibility and to not have the API explicitly refer to PyAV.